apps/cow-fi/pages/products.tsx

```diff
@@ -0,0 +1,80 @@
+import { Color } from '@cowprotocol/ui'
+import Layout from '@/components/Layout'
+import { PRODUCT_LIST } from '@/data/home/const'
+import { Link, LinkType } from '@/components/Link'
+import { EventCategories, sendEventHandler } from '@cowprotocol/analytics'
+import SVG from 'react-inlinesvg'
+
+import {
+  PageWrapper,
+  ContainerCard,
+  ContainerCardSection,
+  TopicList,
+  TopicCard,
+  TopicImage,
+  TopicTitle,
+  TopicDescription,
+  SectionTitleWrapper,
+  SectionTitleText,
+  TopicCardInner,
+} from '@/styles/styled'
+
+export default function Page() {
+  return (
+    <Layout bgColor={Color.neutral90}>
+      <PageWrapper>
+        <ContainerCard bgColor={Color.neutral100} touchFooter>
+          <ContainerCardSection>
+            <SectionTitleWrapper color={Color.neutral0} maxWidth={1200} margin="100px auto">
+              <SectionTitleText lineHeight={1.6} lineHeightMobile={1.8} fontSizeMobile={28}>
+                CoW DAO develops the <span className="wordtag-orange">most user-protective</span> products in DeFi – so
+                you can <span className="wordtag-purple">do more</span> with{' '}
+                <span className="wordtag-blue">less worry</span>
+              </SectionTitleText>
+            </SectionTitleWrapper>
+
+            <TopicList columns={2}>
+              {PRODUCT_LIST.map((topic, index) => (
```

Comment:
This code seems to be duplicated in `cow-fi/pages/index.tsx`, we probably should move it to a component and reuse

Reply:
That's what @anxolin initially had done. In my opinion I consider it "content" and would prefer to keep it outside a component.

Reply:
I don't get it, why content can live int the components? 

I mean, the components can help us to not duplicate the code. Here we are duplicating a lot of lines of code. I don't see the point.

Reply:
Because then we are mixing pure components and pages (considered content). In my opinion it's best to keep it separated as a principle.